src/dappsExplorer/search/DAppsExplorerScreenSearch.test.tsx

```diff
@@ -345,5 +346,39 @@ describe(DAppsExplorerScreenSearch, () => {
       // Names display correctly in the all dapps section
       expect(within(allDappsSection).getByText(dappsList[0].name)).toBeTruthy()
     })
+
+    it('triggers events when searching', async () => {
+      jest.useRealTimers()
```

Comment:
I'm proposing a patch to avoid real timers: https://github.com/valora-inc/wallet/pull/3628